### PR TITLE
[UPDATE] pluralize.md

### PR DESCRIPTION
**Pull request by atomiks**
## Description

- Allows for autopluralization with currying
- Allows any number. `-1` and `1` are considered "singular". Actually should -1 be plural? I don't really know. It doesn't make sense to pluralize negatives, but I feel like it shouldn't throw an error at all. It also coerces strings etc to a number to make life easier.

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Chalarangelo** [comment]
Pluralizing negatives is valid imo. I mean you might want to say minus that many things, so yeah, no problem with that.

File: snippets/pluralize.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
-# pluralize
+### pluralize
```
Comment (Chalarangelo) [inline snippets/pluralize.md]: This was breaking the website before! 😆

**Comment by kriadmin** [comment]
@Chalarangelo @atomiks I think should exist as an separate snippet 😕 . Also if `num` is 0 it should return a empty string. This is much more difficult to understand than the original one.

**Comment by Chalarangelo** [comment]
@kriadmin I sincerely disagree, the snippet is barely any longer and it serves a very practical purpose.

**Comment by skatcat31** [comment]
Is the auto pluralization needed?

**Comment by atomiks** [comment]
@kriadmin `0` is a must.

For example, if you want to pluralize `0 votes` or `0 comments`, how would you do that? Those are very common in apps

@skatcat31 yes, if you want to import a dictionary of plurals and then just use the function once over and over without needing to think about the 3rd argument. Very useful imo

**Comment by skatcat31** [comment]
Yeah I think I'm just thinking that the closure could somehow be avoided( probably not without greatly harming readability )

**Pull request merged**